quizmaster-client-app/src/components/Common/AboutModal.tsx

```diff
@@ -19,19 +36,71 @@ export default function AboutModal(props: any) {
       open={props.modalOpen}
       onClose={props.onClose}
       aria-labelledby="simple-dialog-title"
+      maxWidth="md"
     >
-      <DialogTitle id="simple-dialog-title">Quiz.Coffee</DialogTitle>
-      <div className={classes.aboutModalPaper}>
-        <Typography variant="body2" color="textSecondary" align="center">
+      <DialogTitle id="simple-dialog-title" className={classes.aboutHeader}>
+        Quiz.Coffee - The Perfect App for Coffee Break Quizzes
+      </DialogTitle>
+      <DialogContent>
+        <DialogContentText
+          id="alert-dialog-description"
+          className={classes.createdBy}
+        >
           Created by Calum Tomeny, Richard Lewis and Tim Loudon
+        </DialogContentText>
+        <Typography variant="h4" className={classes.faqHeader}>
+          Frequently Asked Questions
         </Typography>
+        <div>
+          <Typography variant="h5" className={classes.faqQuestion}>
+            Is this a silent quiz app?
+          </Typography>
+          <Typography variant="body2" className={classes.faqAnswer}>
+            {`No. Quiz.Coffee is an app for socialising!
+            Therefore it should be used in combination with web-meet tools such as Zoom, Google Meet, Teams etc;
```

Comment:
Adding open source option as the first option.

```suggestion
            Therefore it should be used in combination with web-meet tools such as Jitsi, Zoom, Google Meet, Teams etc;
```